Bottom: 03e08950f1e86d9bd2c3351e0642afbf6737ab88
Top:    de6a6b78e284bea8263f34d5d05369232c7c1166
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-08 21:29:27 +0000

Revert "Just make a black background."

This reverts commit cf6c95ade3be149a5a8659bd13a80eedc4092fd2.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index a8751ad..63c4df5 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2090,9 +2090,9 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glEnable (GL_STENCIL_TEST);
   glEnable (GL_DEPTH_TEST);
   glDepthFunc (GL_ALWAYS);
-  glClearColor (port->bg_color.red / 65535.,
-                port->bg_color.green / 65535.,
-                port->bg_color.blue / 65535.,
+  glClearColor (port->offlimits_color.red / 65535.,
+                port->offlimits_color.green / 65535.,
+                port->offlimits_color.blue / 65535.,
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
@@ -2196,7 +2196,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* glStencilFunc (GL_GREATER, 1, 1); */           /* Draw only where stencil buffer is 0 */
 
   glDepthMask (GL_FALSE);
-#if 0
   if (global_view_2d) {
     glBegin (GL_QUADS);
     glVertex3i (0,             0,              0);
@@ -2227,7 +2226,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     }
     glEnd ();
   }
-#endif
 
   glDepthMask (GL_TRUE);
